.editorconfig

```diff
@@ -14,3 +14,9 @@ insert_final_newline = false
 
 [website/blog/*.md]
 trim_trailing_whitespace = false
+
+[tests/{**/__snapshots__/*, tests/format/**/*}]
+trim_trailing_whitespace = false
+
+[tests/format/**/jsfmt.spec.js]
+trim_trailing_whitespace = true
```

Comment:
I think it would be ideal to propose these changes in the upstream Prettier project.

Reply:
Agreed - these changes aren't specific to prettierX.